server/templates/registration/logged_out.html

```diff
@@ -1,6 +0,0 @@
-{% extends "server/template.html" %}
-
-{% block body %}
-  <p>התנתקת בהצלחה!</p>
-  <a href="{% url 'login'%}">לחץ כאן להתחבר שוב.</a>
-{% endblock %}
```

Comment:
Again, isn't this necessary (look at comment from login.html)

Reply:
fixed